apps/web/client/src/app/project/[id]/_components/right-panel/chat-tab/chat-input/index.tsx

```
@@ -182,6 +195,53 @@ export const ChatInput = observer(() => {
+
+            let screenshotData = null;
+            let mimeType = 'image/jpeg';
+            
+            for (const frame of framesWithViews) {
+                try {
+                    if (!frame.view || typeof frame.view.captureScreenshot !== 'function') {
+                        continue;
+                    }
+
+                    const result = await frame.view.captureScreenshot();
+                    if (result && result.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)